network/src/protocols/wire/proxy_protocol.rs

```diff
@@ -0,0 +1,369 @@
+// Copyright (c) The Libra Core Contributors
```

Comment:
let's move this into netcore, since it's pretty generic and not libranet specific

Reply:
Moved.  I'm not really clear on all the location distinctions still, but I'll go with it